manifests/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
I don't think this will work for the Relayers. Each Relayer will have to have it's own identity which will be represented by it's HSM key (in Azure Key Vault). The tricky part is that each node needs to know which KeyVault/Key to use and this must be coordinated across the entire cluster. Similarly, the pod that does get access, should have access to a Managed Service Identity (MSI) in that AKS cluster. AKS manages a pool of identities (see [aad-pod-identities](https://github.com/Azure/aad-pod-identity#role-assignment)) and will control which pods have access to which identities. With the identity, the pod can then authenticate to KeyVault. This is all managed at the infrastructure level so no secrets need to be stored/passed.